app/views/ControllerInputsView.js

```diff
@@ -0,0 +1,182 @@
+var channel_config = remote.require('./config/vehicle-config');
+var picpilot_conf = remote.require('./config/picpilot-config');
+
+module.exports = function (Marionette) {
+
+  return Marionette.ItemView.extend({
+    template: Template('ControllerInputsView'),
+    className: 'controllerInputsView',
+
+    ui: {
+      front23: '.circle.front.twothree',
+      front14: '.circle.front.onefour',
+      ch1_text: '.value.one',
+      ch2_text: '.value.two',
+      ch3_text: '.value.three',
+      ch4_text: '.value.four',
+      auto_pilot_btn: '.auto-pilot',
+    },
+
+    initialize: function () {
+      this.throttle = 0;
+      this.roll = 0;
+      this.pitch = 0;
+      this.yaw_rudder = 0;
+      this.auto_on_off = true;
+      this.control_off = false;
+      this.is_scaled = false;
+      this.type = picpilot_conf.get('vehicle_type');
+    },
+    
+    rcIsOff: function(data){
+      var current_vehicle = channel_config.get(this.type);
+      var ch1 = data[current_vehicle['throttle']];
+      var ch2 = data[current_vehicle['roll']];
+      var ch3 = data[current_vehicle['pitch']];
+      var ch4 = data[current_vehicle['rudder_yaw']];
+      var ch5 = data[current_vehicle['autopilot_on_off']];
+      return (ch1 == -10000 || ch2 == -10000 || ch3 == -10000 || ch4 == -10000 || ch5 == -10000);
+    },
+    
+    aircraftChannelCallback: function(data){
+      var current_vehicle = channel_config.get(this.type);
+      var ch1 = data[current_vehicle['throttle']];
+      var ch2 = data[current_vehicle['roll']];
+      var ch3 = data[current_vehicle['pitch']];
+      var ch4 = data[current_vehicle['rudder_yaw']];
+      var ch5 = data[current_vehicle['autopilot_on_off']];
+      this.is_scaled = data['channels_scaled'];
+      // if RC controller is OFF
+      if(this.rcIsOff(data)){
+        this.control_off = true;
+        this.ui.front14.css({"margin-left": "43%","margin-top":"43%",
+                               "background-color": "rgba(244,122,122,1)",
+                               "border-color": "rgba(244,122,122,1"});
+        this.ui.front23.css({"margin-left": "43%","margin-top":"43%",
+                               "background-color": "rgba(244,122,122,1)",
+                               "border-color": "rgba(244,122,122,1"});
+      }else{
+        this.ui.front23.css("background-color", "rgba(32,32,32,1)");
+        this.ui.front14.css("background_color", "rgba(32,32,32,1)");
+        this.control_off = false;
+      }
+      this.setThrottle(ch1);
+      this.setRoll(ch2);
+      this.setPitch(ch3);
+      this.setYawRudder(ch4);
+      this.setOnOff(ch5);
+    },
+    
+    onRender: function(){
+      this.aircraft_channel_callback = this.aircraftChannelCallback.bind(this);
+      TelemetryData.addListener('aircraft_channels', this.aircraft_channel_callback);
+    },
+    
+    setThrottle(val){
+      if(val != null){
+        if(this.control_off){
+          this.ui.ch1_text.text("");
+        }else{
```

Comment:
space. Try to fix all your if and else statements this way (add sufficient space in the curly brackets. ie. ` }else{` -> `} else {`

and `if(val != null){` -> `if (val != null) {`